### PR TITLE
Hip

### PR DESCRIPTION
**Pull request by cthorpe123**
Merge hyperon production into main release.

### CODE REVIEW

File: src/Interaction.cc [hunk 1]
```diff
@@ -397,7 +397,7 @@ void Interaction::get_NN_xsec( double Ek, double &resii, double &resij )
 // hyperon nucleon scatter                                                              
 
 /* Argument list:                                                                      
-Ek hyperon KE in nucleon rest frame                                                    
+Ek hyperon KE in target rest frame
```
Comment (KajetanNiewczas) [inline src/Interaction.cc]: target is always nucleon

**Comment by cthorpe123** [comment]
Hi Kajetan,

I've made that file readable in vim, not sure what happened to it. I can either send that file to you or just push the change to my fork.

The c++17 and removal of the versioning was needed to get it to build on my machine.

The comment about target is to avoid prevent confusion over whether I'm working in the nucleons rest frame or the rest frame of the entire nucleus.

Thanks,
Chris
________________________________

**Comment by KajetanNiewczas** [comment]
> Hi Kajetan, I've made that file readable in vim, not sure what happened to it. I can either send that file to you or just push the change to my fork. The c++17 and removal of the versioning was needed to get it to build on my machine. The comment about target is to avoid prevent confusion over whether I'm working in the nucleons rest frame or the rest frame of the entire nucleus. Thanks, Chris
> […](#)

Sorry for a lot of mess everywhere. I'm not used to pull requests via GitHub website. I wanted to add more comments before sending but now I have not idea what you see and what you don't.

I need to think what to do about c++17. I already forgot what was the problem, but it's weird. Wasn't that caused by something else in the end? I remember something about trying to run cmake.

File: src/Interaction.cc (hunk 1 above)
Comment (KajetanNiewczas) [reply src/Interaction.cc]: I assume your xsec is hyperon-nucleon, so it should be in nucleon rest frame. Same as all other xsecs here.

Comment (KajetanNiewczas) [reply src/Interaction.cc]: Okay, I see now. A shortened version of the code on GitHub misled me. Now I understand.

**Pull request merged**